summary.r

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env Rscript
+# Author: OLP/2022
+# Usage: summary for SARS-CoV-2 run, only for PatientSamples, input: one parameter
+# ie. Rscript /. . . ./summary.r UT-A01290-220416
+
+library("plyr", warn.conflicts = F)
+library("dplyr", warn.conflicts = F)
+library("rbin")#
+library("tidyverse")
+library("lubridate", warn.conflicts = F)
+library("tibble")
+library("stringr") 
+require("seqinr"); 
+library("XML")
+#===============================================
+args = commandArgs(trailingOnly=T)
+# args[1]<-c('UT-A01290-220416'); # only to works on script preparation
+date<-ymd(substr(args[1], 11, 16));runPath <-paste('/Volumes/NGS/Analysis/covidseq',args[1],sep="/"); head(date)
+
+#=============================================
+ss1 = paste(runPath,"covidseq_output/SampleSheet_Intermediate.csv",sep ="/"); 
+ss2 = paste(runPath,"covidseq_output/lineage/combined_lineage_report.csv",sep ="/"); 
+ss3 = paste(runPath,"covidseq_output/Logs_Intermediates/VirusDetection/kmer/Summary.tsv",sep ="/");
+ss5 = paste(runPath,"covidseq_output/Logs_Intermediates/FastqGeneration/Reports/RunInfo.xml",sep ="/");
+df1 = read.csv(ss1); # SampleSheet
+df2 = read.csv(ss2); # combined_llineage_report
+df3 = read.table(file = ss3, sep = '\t', header = T); # sc2_amplicons 
+
+#=============================================
+# collectin basic info from samplesheet
+t<-.5;
+instrument_type<-df1[4,2];cat("instrument_type:", instrument_type,"\n");Sys.sleep(t)
+wetlab_assay<-df1[5,2]; cat("wetlab_assay:", wetlab_assay,"\n");Sys.sleep(t)
+pangolin_version<-df2[1,9];cat("pangolin_version :",pangolin_version,"\n");Sys.sleep(t)
+pangoLEARNversion<-df2[1,8];cat("pangoLEARNversion :",pangoLEARNversion,"\n");Sys.sleep(t)
```

Comment:
With the updated Pangolin tool (v 4.0+), the lineage report does not contain 'pangoLEARNversion' column. You might want to consider checking the lineage report generated by the the newer version and add any other additional columns that might be worth reporting.

Reply:
Is this true with Illumina's lineage container as well?

Reply:
Illumina's lineage tools still doesn't have the updated pangolin container but once it does, I think the column headers will change and it might be good to make the code robust to such changes. Alternatively, it will need to be modified later after the update.